tools/walletextension/api/staticOG/index.html

```diff
@@ -11,25 +11,58 @@
 </head>
 
 <body>
+        <div class="container-fluid">
+            <img src="obscuro.svg" alt="Obscuro" class="logo">
+        </div>
 
-<h2>◠. Obscuro Gateway Demo! ◠.</h2>
-
-
-<button class="btn btn-primary btn-lg btn-block mb-3" id="join" style="display: none">Join</button>
-<button class="btn btn-primary btn-lg btn-block mb-3" id="revokeUserID" style="display: none">Revoke UserID</button>
-<p id="status"></p>
-<table id="accountsTable" border="1">
-    <thead>
-    <tr>
-        <th>Accounts</th>
-        <th>Added to Obscuro Gateway</th>
-    </tr>
-    </thead>
-    <tbody id="tableBody">
-    <!-- Rows will be added here -->
-    </tbody>
-</table>
-
-<div id="versionDisplay" style="position: fixed; bottom: 10px; right: 10px;"></div>
+        <div class="spinner" id="spinner"></div>
+
+<div class="begin-box" id="begin-box">
+    <h2>Welcome to the Obscuro Gateway</h2>
+    <form>
+        <div id="information">A 3 click process to setup encrypted communication between MetaMask and the Obscuro Network.
+            Upon hitting Begin, you'll be asked to:
+            <ol>
+                <li>
+                    Add the Obscuro Testnet network as a new network to MetaMask. Check the Network URL base is https://testnet.obscu.ro and the chain ID is 443 and hit approve  
+                </li>
+                <li>
+                    Allow MetaMask to switch networks to the Obscuro Testnet network
```

Comment:
We should probably mention that users need to select which accounts they want to connect? (for the first time users)